en/release-notes.md

```diff
@@ -25,6 +25,56 @@ sudo add-apt-repository -yu ppa:maas/next
 sudo apt install maas
 ```
 
+## 2.3.0 (beta1)
+
+### Hardware Testing
+
+This release overhauls the visibility of hardware test results, necessitating
+significant changes across the MAAS web UI:
+
+- **Machine/Nodes Listing**.
+  This page now includes the progress and status of hardware tests. Status
+  shows  when a test is pending, running, successful, or when a test has
+  failed. Additionally, machines can now be filtered by architecture.
+
+- **Machine Details**:
+
+    - **Machine summary** includes new details about the different components
+      being tested, such as CPU, memory and storage.
+
+    - **Hardware tests** have been completely redesigned, showing a list of
+      test results per component and the ability to view more details about the
+      test itself.
+
+    - **Settings** is a new page that enables you to edit a node's
+      configuration directly.
+
+
+### Rack Controller Deployment
+
+This release adds the ability to deploy any machine with a rack controller via the API.
+
+
+### Other UI improvements
+
+The subnets page now includes a *DHCP* column and in-line editing has been
+removed from both the *Space* and *VLAN* pages. You will now also be warned when
+changing a subnet's mode to and from either unmanaged or managed. 
```

Comment:
I would have said "mode between managed and unmanaged" Those are the only 2, yes?

Reply:
yeah, you're right. thanks!